src/app/helpers/whitelistApis.js

```diff
@@ -996,15 +996,15 @@ const API_LIST = {
         ROLE.BOOK_CREATOR
       ]
     },
-    '/action/dialcode/v1/reserve/:do_id': {
+    '/action/dialcode/v3/reserve/:do_id': {
       checksNeeded: ['ROLE_CHECK'],
       ROLE_CHECK: [
         ROLE.CONTENT_CREATOR,
         ROLE.COURSE_CREATOR,
         ROLE.BOOK_CREATOR
       ]
     },
-    '/action/dialcode/v1/process/status/:processId': {
+    '/action/dialcode/v3/process/status/:processId': {
```

Comment:
This requires new collection-editor which is calling this end-point.

Otherwise, we should only do the changes in the Kong. We will be calling same as before(dialcode/v1/process). 

But on Kong, it will change to `dialcode/v3/process`